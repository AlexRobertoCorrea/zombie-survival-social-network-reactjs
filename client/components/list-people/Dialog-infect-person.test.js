import React from 'react';

import DialogInfectPerson from './Dialog-infect-person';

describe('<DialogInfectPerson />', () => {
  describe('Renders infected dialog correctly', () => {
    const open = true;
    const person = {};
    const handleCloseInfect = () => {};
    const handleInfectPerson = () => {};
    
    const dialogInfectPerson = shallow(<DialogInfectPerson
      open={open}
      person={person}
      handleCloseInfect={handleCloseInfect}
      handleInfectPerson={handleInfectPerson}
    />);
    
    it('Ensure that dialogInfectPerson works as expected', () => {
      expect(dialogInfectPerson).toMatchSnapshot();
    });
  });
});
