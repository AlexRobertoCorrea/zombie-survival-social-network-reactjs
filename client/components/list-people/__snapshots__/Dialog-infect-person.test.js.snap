// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogInfectPerson /> Renders infected dialog correctly Ensure that dialogInfectPerson works as expected 1`] = `
<SimpleDialog
  options={
    Object {
      "content": <WithStyles(DialogContentText)
        id="alert-dialog-description"
      >
        Are you sure you want mark 
         as infected?
      </WithStyles(DialogContentText)>,
      "leftButtonLabel": "Cancel",
      "leftButtonOnClick": [Function],
      "onClose": [Function],
      "open": true,
      "rightButtonLabel": "Accept",
      "rightButtonOnClick": [Function],
      "title": "Infect person",
    }
  }
/>
`;
